### DISCUSSION TITLE
[Bug] TabView Design Challenges

### DISCUSSION THREAD

**Discussion by sattasundar** [xamarin/XamarinCommunityToolkit#1184] [Q&A]
### Description
Unable to find similar TabView design in XCTSample code whichever shown in XCT blog.

![image](https://user-images.githubusercontent.com/62828724/114055560-818a7b80-98ae-11eb-8842-e6b81479089f.png)

### Basic Information
Need to design navbar like below image;
![WhatsApp Image 2021-04-08 at 9 18 19 PM](https://user-images.githubusercontent.com/62828724/114057197-fca06180-98af-11eb-8bce-cb018ecb4eed.jpeg)

Need suggestion if any; how to achieve this using XCT.

**Comment by Cfun1** [comment]
Hi, thanks for your suggestion but this doesn't looks like a bug it's either an enhancement or a question ([discussion](https://github.com/xamarin/XamarinCommunityToolkit/discussions) section).
Could you please provide the link to the "xct blog" you mentioned ?

**Reply by AndreiMisiukevich** [reply]
I converted it to the conversation. It is certainly not a bug

**Reply by sattasundar** [reply]
Thanks for the response. 

Here is the blog link 
https://devblogs.microsoft.com/xamarin/xamarin-community-toolkit/

**Comment by Cfun1** [comment]
The first sample of [custom Tabs](https://github.com/xamarin/XamarinCommunityToolkit/blob/main/samples/XCT.Sample/Pages/Views/TabView/CustomTabsPage.xaml) looks similar to what you are trying to achieve have you tried to adapt it ?
